core/src/main/resources/templates/error.vm

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
This is the default error page. A standard Spring Boot feature when Apache Velocity is available.